modules/monitoring-resizing-a-persistent-storage-volume.adoc

```diff
@@ -0,0 +1,203 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="resizing-a-persistent-storage-volume_{context}"]
+= Resizing a persistent storage volume
+
+{product-title} does not support resizing an existing persistent storage volume using StatefulSets, even if the underlying `StorageClass` used supports persistent volume sizing.
+Therefore, even if you update the `storage` field for an existing persistent volume claim (PVC) with a larger size, this setting will not be propagated to the associated persistent volume (PV).
+
+However, resizing a PV is still possible by using a manual process. If you want to resize a PV for a monitoring component such as Prometheus, Thanos Ruler, or Alertmanager, you can update the appropriate config map where the component is configured, patch the PVC, and then delete and orphan the pods.
+Orphaning the pods recreates the StatefulSet immediately and automatically updates the size of the volumes mounted in the pods with the new PVC settings.
+No service disruption occurs during this process. 
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* *If you are configuring core {product-title} monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
```

Comment:
since we are talking about `Resizing a persistent storage volume` here, we'd better mention we've already attached PVs for components,and we want to resize the storage size, so we'd better add one pre-condition:
`You have configured PVCs for core OpenShift Container Platform monitoring components`

Reply:
Added `You have configured at least one PVC for core {product-title} monitoring components.`